api/desecapi/models.py

```diff
@@ -231,10 +232,17 @@ class RenewalState(models.IntegerChoices):
     owner = models.ForeignKey(User, on_delete=models.PROTECT, related_name='domains')
     published = models.DateTimeField(null=True, blank=True)
     minimum_ttl = models.PositiveIntegerField(default=get_minimum_ttl_default)
-    renewal_state = models.IntegerField(choices=RenewalState.choices, default=RenewalState.FRESH)
+    renewal_state = models.IntegerField(choices=RenewalState.choices, default=RenewalState.IMMORTAL)
     renewal_changed = models.DateTimeField(auto_now_add=True)
     _keys = None
 
+    def __init__(self, *args, **kwargs):
+        name = kwargs.get('name')
+        if name and kwargs.get('renewal_state') is None \
+                and any(name.endswith(f'.{lps}') for lps in settings.LOCAL_PUBLIC_SUFFIXES):
```

Comment:
move `any(name.endswith(f'.{lps}') for lps in settings.LOCAL_PUBLIC_SUFFIXES)` into a property of `Domain`

Reply:
done